test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -224,15 +224,15 @@ public String getID() {
     // Removal of the JVM option 'UseCGroupMemoryLimitForHeap' is required with JVM environments >= 10. This option
     // is supplied by default by the operators. We cannot 'deactivate' it using the XX:- counterpart as it is unrecognized.
     private String[] getSegmentStoreJVMOptions() {
-        return new String[]{"-XX:+UseContainerSupport", "-XX:+IgnoreUnrecognizedVMOptions"};
+        return new String[]{"-XX:+UseContainerSupport", "-XX:+IgnoreUnrecognizedVMOptions", "-XX:MaxDirectMemorySize=5g", "-Xmx1024m"};
```

Comment:
Max 5GB for DM; 1GB for heap. In line 134 you have an allocation of  6GB, which doesn't leave any room for the runtime.

Should we set DM to 4GB, heap to 1GB, and Cache to 2GB (I remember the cache was defaulted to 4GB - should we adjust that too?)